Command/TestShortenerCommand.php

```diff
@@ -31,48 +46,47 @@ protected function configure()
 
         $this
             ->setDescription('Tests shortener')
-            ->setName('mremi:url-shortener:test');
+            ->setName('mremi:url-shortener:test')
+            ->addOption(
+                'provider',
```

Comment:
Should be `providers`

Reply:
hi, I  think it is enough to  test a specific provider, not a group of providers, we already have codes to test all providers.